spec/requests/jams_request_spec.rb

```diff
@@ -55,5 +55,26 @@
         end.to_not change(Jam, :count)
       end
     end
+
+    context "user is unauthenticated" do
+      it "does not associate a user with a created jam" do
+        post room_jams_path(room), params: { jam: { bpm: '100', file: file } }
```

Comment:
at this point it makes sense to introduce a let or a DSL method for `uploads a jam` since it's in a bunch of specs